### DISCUSSION TITLE
question about model.py

### DISCUSSION THREAD

**Discussion by wooodswu** [lululxvi/deepxde#1257] [General]
<img width="666" alt="image" src="https://user-images.githubusercontent.com/87766834/232685901-be0da24a-8bf2-4cda-b510-a84a832bbbff.png">
Need your help! I have a question about this “losses_fn", except in the definition of “outputs_losses”, i cannot find this function. When I want to change the loss function, and using the following:
/*code
net = dde.maps.DeepONetCartesianProd(
    [101,30,30,101], [1,30,30,101], "relu", "Glorot normal")
model2 =dde.Model(data, net)
model2.compile("adam",
    lr=3e-4,
    loss=“mse",
    metrics=["l2 relative error"],
)
*/
dde.Model.compile(): loss, by my understanding, is just the “loss_fn”(here is just mse) in the “losses_fn", and I cannot find the  “losses_fn” in the whole package. 
My question: could I change this losses_fn? is this the loss function I need to define? Or in the somewhere of the model, where can  I change the loss?

**Comment by lululxvi** [comment]
https://github.com/lululxvi/deepxde/blob/master/deepxde/losses.py
